util-core/src/main/scala/com/twitter/util/Batcher.scala

```diff
@@ -0,0 +1,30 @@
+package com.twitter.util
+
+import java.util.logging.{Level, Logger}
+
+import scala.collection.mutable
+
+/** Provides a clean, lightweight interface for controlling a BatchExecutor
```

Comment:
So passing the BatchExecutor as a constructor parameter was your idea to reduce the clutter of having to pass all of the BatchExecutor's constructor params through two consecutive constructor methods.  And no one's creating Batchers directly, they're getting them from Future.batched.

So I guess I'm wondering what you think now: should we back off having the BatchExecutor as a parameter to the Batcher constructor, or should we just make the constructor private[util] and leave it at that.